circle.yml

```diff
@@ -1,6 +1,9 @@
 machine:
   ruby:
     version: 2.4.1
+  pre:
+    - sudo apt-get install libfontconfig
+    - sudo curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-2.1.1
```

Comment:
TECHNICALLY this is a bit odd since you use prebuilt in Docker but then use the standard phantomjs here. Practically it probably doesn't matter, but worth pointing out. 

Also curious if https://github.com/Medium/phantomjs#continuous-integration instructions on caching can be applied for CircleCI.